examples/next/pages/ui/components/account-settings/delete-user/index.page.tsx

```diff
@@ -16,8 +16,9 @@ Amplify.configure(awsExports);
 export default function App() {
   return (
     <Authenticator>
-      {({ signOut }) => (
+      {({ signOut, user }) => (
         <Card width="800px">
+          <h1>Hello {user.attributes.email}</h1>
```

Comment:
Just curious how this will prevent accidental deletion?

Reply:
I don't think it does, maybe it's just for smoke testing that it's deleted?

Reply:
If the user was deleted, then this would just break right?

Reply:
It prevents accidental deletion by telling the user which account they are using 😄 

I accidentally deleted because I didn't know that test account was cached in the browser 😂

Reply:
But that won't actually prevent deletion right?

Reply:
@calebpollman right